### PR TITLE
Include an example for using DecodeMulti in the Readme

### LINKED ISSUE #67: Document using multiple codecs in the readme
**Is your feature request related to a problem? Please describe.**
> I'd like the ability to more easily rotate keys by being able to add a list of valid decryption keys.

**Describe the solution you'd like**
>Change 
`New(hashKey, blockKey []byte)` to 
`New(hashKey, blockKey []byte, decodeKeys ...[]byte)` and add a check that if a cookie cannot be decoded with the hashKey to try with the decode keys until one matches or all fail. This would allow users to have some logic like 
`New(todaysKey(), nil, tomorrowsKey(), yesterdaysKey())` and rotate keys automatically even in a distributed environment. It would also preserve backwards compatibility with `New(hashKey, nil)`.

**Describe alternatives you've considered**
> There's the obvious, just don't rotate the keys. Or coordinate a failover mechanism by chaining decodes using multiple secure cookies. This seems like a more straightforward approach however.

### PR DESCRIPTION
**Pull request by tflyons**
Fixes #67

**Summary of Changes**

1. Add a section to the README to give an example of using the EncodeMulti/DecodeMulti functions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,66 @@ registered first using gob.Register(). For basic types this is not needed;
...
+### Key Rotation
+Rotating keys is an important part of any security strategy. The `EncodeMulti` and
+`DecodeMulti` functions allow for multiple keys to be rotated in and out.
+For example, let's take a system that stores keys in a map:
+
+```go
+var cookies = map[string]*securecookie.SecureCookie{
+	"previous": securecookie.New(
+		securecookie.GenerateRandomKey(64),
+		securecookie.GenerateRandomKey(32),
```
Comment (elithrar) [inline README.md]: Best to have in-line comments here warning against this: people copy paste the code & stop reading further 🙃

File: README.md [hunk 2]
```diff
@@ -77,6 +77,66 @@ registered first using gob.Register(). For basic types this is not needed;
...
+	),
+	"current": securecookie.New(
+		securecookie.GenerateRandomKey(64),
+		securecookie.GenerateRandomKey(32),
+	),
+}
+```
+It's important to point out that keys stored in a map would not be persisted
```
Comment (elithrar) [inline README.md]: Make this a comment as above.

File: README.md (hunk 1 above)
Comment (tflyons) [reply README.md]: Good call, fixed

**Comment by tflyons** [comment]
Any update on this?

**Comment by elithrar** [comment]
Thanks!

**Pull request merged**